cli/internal/lockfile/berry_lockfile.go

```diff
@@ -104,3 +116,21 @@ func DecodeBerryLockfile(contents []byte) (*BerryLockfile, error) {
 	prettyLockFile := BerryLockfile(yarnSplitOutEntries(lockfile))
 	return &prettyLockFile, nil
 }
+
+func yarnSplitOutEntries(lockfile map[string]*YarnLockfileEntry) map[string]*YarnLockfileEntry {
+	prettyLockfile := map[string]*YarnLockfileEntry{}
+	// This final step is important, it splits any deps with multiple-resolutions
+	// (e.g. "@babel/generator@^7.13.0, @babel/generator@^7.13.9":) into separate
+	// entries in our map
+	// TODO: make concurrent
+	for key, val := range lockfile {
+		if strings.Contains(key, ",") {
+			for _, v := range strings.Split(key, ", ") {
+				prettyLockfile[strings.TrimSpace(v)] = val
+			}
+		} else {
+			prettyLockfile[key] = val
+		}
```

Comment:
I believe this is doing a bunch of unnecessary work.
```suggestion
for _, v := range strings.Split(key, ", ") {
  prettyLockfile[v] = val
}
```

Reply:
I think you're correct. I was just moving this code from the yarn file since it's no longer needed there.